### DISCUSSION TITLE
Command Palette Not Finding Forked Repositories

### DISCUSSION THREAD

**Discussion by farkastime** [community/community#102125] [Repositories]
### Select Topic Area

Product Feedback

### Body

Looks to me like the Command Palette does not recognize forks as repositories, so they will never be suggested in any scope. Need I do something to these forks to make them indexed? Or is this better flagged as a feature / bug request?

**Comment by ghostinhershell** [comment]
Hi @tfarkas86, 

Thanks so much for writing in and participating in the Accessibility community. From checking our Docs it does look like forks aren't indexed when searching:
- https://docs.github.com/en/get-started/accessibility/github-command-palette#searching-with-the-github-command-palette

I'll submit this over to the Accessibility team as a feature request and update you here with any updates that I receive from the team.

**Reply by farkastime** [reply]
Thanks a bunch, @ghostinhershell! I love this approach to searching content, so adding forks would really help us out. Hopefully others would find that a useful capability. Cheers!

**Reply by Sackbuoy** [reply]
Would love this feature as well! I love the command palette but of our repositories are forks
